src/Icons/src/Twig/IconFinder.php

```diff
@@ -38,11 +38,7 @@ public function icons(): array
         foreach ($this->files($this->twig->getLoader()) as $file) {
```

Comment:
Should we exclude the Bundle ?

Reply:
What bundle?

Reply:
At this point, i think Twig loader has been registered with all the bundles of the app. So it contains all the files in "vendor/acme/*-bundle/Resources/templates" and we should (imho) not go there to extract things. 

Imagine a twig file there with a list of icons (think any template using JS icons)... the probability to have 'bi:check' string is far from null. 

We even could see this as a vector "attack", a single file in twig from a 3rd party bundle would make you downlaod icons locally / generate a lot of download. 

Oh and i'll check something about that we may have a problem there.


So here i'd like us to only look in userland files :)

Reply:
What if you had a bundle of that uses icons that you wanted to use? I prefer to keep it simple and adjust when/if we come across these type of problems.